### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by jacobvjk**
Tiny corrections in markdown, to have the backslashes displayed

### CODE REVIEW

File: _posts/2020-07-27-config-project-settings/config-project-settings.Rmd [hunk 1]
```diff
@@ -60,7 +59,7 @@ Project wide settings should be set in the file "project_settings.yml", prior to
 - Please define `project_location_ext` by copying the full path to the folder, in which you want to store data (including the results) related to this particular evaluation. During the run of the code, the a project folder with several sub-folders will be created here.
 
 - Please consider: 
-  - copying in a path from windows requires to change all "\" to "/" (e.g. "C:/Users/Desktop/project_folder/).
+  - copying in a path from windows requires to change all "\\" to "/" (e.g. "C:/Users/Desktop/project_folder/).
```
Comment (maurolepore) [inline _posts/2020-07-27-config-project-settings/config-project-settings.Rmd:62]: Good catch. This is a tricky one. I'll play a bit locally and push the fix.

File: _posts/2020-07-27-config-project-settings/config-project-settings.Rmd [hunk 2]
```diff
@@ -12,7 +12,6 @@ output:
 categories:
   - programming
   - PACTA_analysis
-preview: preview.jpg
```
Comment (maurolepore) [inline _posts/2020-07-27-config-project-settings/config-project-settings.Rmd:15]: This removes the preview image -- which is fine with me but just to check: is the removal intentional?

Would you like to propose a different preview image?

Comment (jacobvjk) [reply _posts/2020-07-27-config-project-settings/config-project-settings.Rmd:15]: woops, that was accidental

File: _posts/2020-07-27-config-project-settings/config-project-settings.Rmd (hunk 1 above)
Comment (maurolepore) [reply _posts/2020-07-27-config-project-settings/config-project-settings.Rmd:62]: Done, your solution worked.

File: _posts/2020-07-27-config-project-settings/config-project-settings.Rmd (hunk 2 above)
Comment (maurolepore) [reply _posts/2020-07-27-config-project-settings/config-project-settings.Rmd:15]: Ok, I'll fix that now.

Comment (maurolepore) [reply _posts/2020-07-27-config-project-settings/config-project-settings.Rmd:15]: Fixed

image

**Pull request merged**